builder/steps/gen-dockerfile/contents/test/detect_setup_test.ts

```diff
@@ -14,6 +14,11 @@
  * limitations under the License.
  */
 
+// Describe the signature of function exported by the `shell-escape` module
+// so the compiler knows what types to expect and return.  This is needed
+// since the `shell-escape` module does not have type definitions available.
+const shellEscape: (args: string[]) => string = require('shell-escape');
```

Comment:
I'm curious what the recommended practice is for writing type signatures for untyped modules. In the case of a module that exports a single function this seems fine but for more complex or widely required modules, inlining type definitions at each require point seems like it would be a lot of duplicated code. Is there a way to specify project local type definitions for external modules?